packages/lib/server/repository/user.ts

```diff
@@ -489,4 +489,13 @@ export class UserRepository {
       },
     });
   }
+
+  static async updateById({ id, data }: { id: number; data: Prisma.UserUpdateInput }) {
+    return prisma.user.update({
+      where: {
+        id,
```

Comment:
Validate the 'id' parameter before using it in the update operation to ensure it is a valid number [important]lskdfjlskd

- Generated by [LGTM](https://lgtm.pro)